share/sql/share.go

```diff
@@ -341,6 +329,27 @@ func (m *ShareMgr) UpdateReceivedShare(ctx context.Context, recvShare *collabora
 
 // Exported functions below are not part of the CS3-defined API, but are used by cernboxcop
 
+// Used by cernboxcop, to include listings with orphans (which cannot be represented in the CS3 Shares)
+func (m *ShareMgr) ListShareModels(ctx context.Context, filters []*collaboration.Filter) ([]model.Share, error) {
```

Comment:
Maybe a name such as `ListModelShares` or `ListMShares` is better than `ListShareModels` ?